### PR TITLE
Frontend improvements 

### PR DESCRIPTION
**Pull request by peyman-mohtashami**

### CODE REVIEW

File: authorizer-app/src/app/components/shared/toolbar/toolbar.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <mat-toolbar color="primary">
   <mat-toolbar-row>
     <div class="logo-wrapper">
-      <img class="logo" src="./assets/radar-logo.svg" alt="RADAR-base Logo" />
+      <img class="logo" src="./assets/chdr_logo.png" alt="CHDR Logo" />
```
Comment (nivemaham) [inline authorizer-app/src/app/components/shared/toolbar/toolbar.component.html:4]: Please roll back these changes. This is the open-source RADAR-base version.

Comment (peyman-mohtashami) [reply authorizer-app/src/app/components/shared/toolbar/toolbar.component.html:4]: Oh, ok. I thought we want to change the repo!

File: authorizer-app/src/app/components/rest-source-authorization/update-rest-source-user.component.ts [hunk 2]
```diff
@@ -83,19 +106,55 @@ export class UpdateRestSourceUserComponent implements OnInit {
...
+
+  onChangeProject(projectId: string) {
+    if(projectId === ''){
+    }else{
+      this.loadAllRestSourceUsersOfProject(projectId)
+    }
+  }
+
+  private loadAllRestSourceUsersOfProject(projectId: string) {
+    this.restSourceUserService.getAllUsersOfProject(projectId).subscribe(
```
Comment (nivemaham) [inline authorizer-app/src/app/components/rest-source-authorization/update-rest-source-user.component.ts:143]: This call is till wrong. Here we should retrieve '/projects/{projectId}/users'.
I will make the changes and commit.

Comment (peyman-mohtashami) [reply authorizer-app/src/app/components/rest-source-authorization/update-rest-source-user.component.ts:143]: Thanks. This is the endpoint which I got confused.

**Review by nivemaham — APPROVED** [review]
LGTM

**Pull request merged**